packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -47,23 +48,41 @@ const MILLI = 1_000_000n;
  * @typedef { WellKnownSpaces & ChainBootstrapSpace & EconomyBootstrapSpace
  * } EconomyBootstrapPowers
  * @typedef {PromiseSpaceOf<{
- *   ammInstanceWithoutReserve: Instance,
- *   ammCreatorFacet: XYKAMMCreatorFacet,
- *   ammGovernorCreatorFacet: GovernedContractFacetAccess<XYKAMMPublicFacet,XYKAMMCreatorFacet>,
+ *   ammFacets: {
+ *     instanceWithoutReserve: Instance,
+ *     creatorFacet: XYKAMMCreatorFacet,
+ *     governorCreatorFacet: GovernedContractFacetAccess<XYKAMMPublicFacet,XYKAMMCreatorFacet>,
+ *     adminFacet: AdminFacet,
+ *     publicFacet: XYKAMMPublicFacet,
+ *   },
  *   economicCommitteeCreatorFacet: import('@agoric/governance/src/committee.js').CommitteeElectorateCreatorFacet,
- *   feeDistributorCreatorFacet: import('../feeDistributor.js').FeeDistributorCreatorFacet,
- *   feeDistributorPublicFacet: import('../feeDistributor.js').FeeDistributorPublicFacet,
+ *   feeDistributorFacets: {
+ *     creatorFacet: import('../feeDistributor.js').FeeDistributorCreatorFacet,
+ *     publicFacet: import('../feeDistributor.js').FeeDistributorPublicFacet,
+ *     adminFacet: AdminFacet,
+ *   },
  *   periodicFeeCollectors: import('../feeDistributor.js').PeriodicFeeCollector[],
  *   bankMints: Mint[],
  *   psmFacets: MapStore<Brand, PSMFacets>,
- *   econCharterStartResult: EconCharterStartResult,
- *   reservePublicFacet: import('../reserve/assetReserve.js').AssetReservePublicFacet,
- *   reserveCreatorFacet: import('../reserve/assetReserve.js').AssetReserveLimitedCreatorFacet,
- *   reserveGovernorCreatorFacet: GovernedAssetReserveFacetAccess,
- *   stakeFactoryCreatorFacet: StakeFactoryCreator,
- *   vaultFactoryCreator: VaultFactoryCreatorFacet,
- *   vaultFactoryGovernorCreator: GovernedContractFacetAccess<{},{}>,
- *   vaultFactoryVoteCreator: unknown,
+ *   econCharterFacets: EconCharterStartResult,
```

Comment:
`SomeContractFacets` is the same as the results of its start function. In this PR,`someContractFacets` is used for the property at `SomeContractStartResult` for the type. Would it be better to be consistent between these cases?
```suggestion
 *   econCharterFacets: EconCharterFacets,
```

Or alternately,
```suggestion
 *   econCharterStartResult: EconCharterStartResult,
```

One argument against the latter is that it might read as the result of `startInstance` which would be inaccurate as that returns `StartedInstanceKit` which has `instance` and `adminFacet` too. Though that brings up another potential confusion: does `SomeContractFacets` include `adminFacet` or not? If not, then `SomeContractFacets` would be misleading.

This all isn't a blocker. Now seems like a good time to come up with consistent terminology for these types and bindings but it could be punted out to a ticket to finalize and implement.

Reply:
In at least some cases, the collection of facets is not exactly the result of `startInstance` but also includes, for example, a governor creator facet.

Reply:
Yeah, It's definitely not the same as `*StartResult`, but it's also not composed of only things that are facets. I'm comfortable with `*Kit`. That naming convention doesn't seem to clash with other variable names.